<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evopedia</class>
 <widget class="QMainWindow" name="Evopedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evopedia Search</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchField"/>
       </item>
       <item>
        <widget class="QComboBox" name="languageChooser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConfigure_Dumps"/>
    <addaction name="actionMap"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionConfigure_Dumps">
   <property name="text">
    <string>Configure Dumps</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
